pymemcache/test/test_client_hash.py

```diff
@@ -354,7 +355,27 @@ class MyClient(Client):
 
         client = HashClient([])
         client.client_class = MyClient
-        client.add_server('host', 11211)
+        client.add_server(('host', 11211))
         assert isinstance(client.clients['host:11211'], MyClient)
 
+    def test_mixed_inet_and_unix_sockets(self):
+        servers = [
+            '/tmp/pymemcache.{pid}'.format(pid=os.getpid()),
+            ('127.0.0.1', 11211),
+        ]
+        client = HashClient(servers)
+        assert set(servers) == set(c.server for c in client.clients.values())
```

Comment:
You can simplify the right-hand side using a set comprehension:

```suggestion
        assert set(servers) == {c.server for c in client.clients.values()}
```

Reply:
:man_facepalming: